react/features/base/react/components/web/MultiSelectAutocomplete.js

```diff
@@ -140,8 +150,11 @@ class MultiSelectAutocomplete extends Component {
                 <MultiSelectStateless
                     filterValue = { this.state.filterValue }
                     isDisabled = { isDisabled }
+                    isLoading = { this.state.loading
+                        && Boolean(this.state.filterValue) }
```

Comment:
what's the check for filterValue here?  if loading can be true without filterValue being truthy, doesn't that mean that isLoading should be true?  if not, is it useful to have loading be true but it not actually be considered 'loading'?